src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx

```diff
@@ -50,7 +51,7 @@ export default class InstagramMediaBlockModal extends React.Component {
       isValid: block.has('id'),
       touched: false,
       updatedDate: block.get('updated_date', new Date()),
-      url: id ? `https://www.instagram.com/p/${id}/` : '',
+      url: '',
```

Comment:
@jalopez13 why? this should show the url when opening modal to edit existing block

Reply:
@efjacobson 

Added `id ? `https://www.instagram.com/p/${id}/` : ''` that shouldn't have been removed.